### PR TITLE
Reviewed the Factories - Getting Started section for language and asc…

### PR DESCRIPTION
**Pull request by supriya-bharadwaj**
…iidoc syntax. Incorporated Preeti and Robin's feedback.

Signed-off-by: Supriya Takkhi <sbharadw@redhat.com>

### What does this PR do?

Fixes lang. and markup issues in *Portable Workspaces -> Getting started with factories*.
File name: `src/main/pages/che-6/portable-workspaces/factories-getting-started.adoc`

Hi @rkratky , can you please review this PR? Thanks!

### CODE REVIEW

File: src/main/pages/che-6/portable-workspaces/factories-getting-started.adoc [hunk 1]
```diff
@@ -9,59 +9,70 @@ folder: che-6/portable-workspaces
 ---
 
 
-A Factory is a template containing configuration to automate generating new Workspace with factory identifier added to IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
+A factory is a template that contains the configuration to automate generation of new Workspace with the factory identifier added to the IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
```
Comment (rkratky) [inline src/main/pages/che-6/portable-workspaces/factories-getting-started.adoc]:
```suggestion
A factory is a template that contains the configuration to automate generation of a new workspace with the factory identifier added to the IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
```

**Review by rkratky — CHANGES_REQUESTED** [review]
@supriya-bharadwaj, use the current 'master' to base your PR on -- this PR is based on an old version of the file.

**Comment by supriya-bharadwaj** [comment]
Hi @rkratky , I tried basing my PR on the latest master. Can you confirm if it is okay now?

**Comment by rkratky** [comment]
> I tried basing my PR on the latest master.

And apparently haven't succeeded.

> Can you confirm if it is okay now?

You can confirm that yourself. Just have a look at the [file that's currently in master](https://github.com/eclipse/che-docs/blob/master/src/main/pages/che-6/portable-workspaces/factories-getting-started.adoc), and you'll see it's different from what you added your edits to.

Honestly, I don't know what you may be doing wrong, but I suggest you just leave this and move on :) I'll fix the file.

**Review by rkratky — APPROVED** [review]
Force-pushed my review based on the current master.

**Comment by rkratky** [comment]
> Just have a look at the file that's currently in master,

Well, I pushed and merged the fixes, so now it's a different file. But you get the idea.

**Pull request merged**